frontend/lib/api/types/meal-plan.ts

```diff
@@ -31,31 +31,44 @@ export interface ListItem {
   quantity?: number;
   checked?: boolean;
 }
+export interface PlanCategory {
+  id: string;
+  name: string;
+  slug: string;
+}
```

Comment:
Does this make our future maintenance more complicated? :thinking: 
Is it creating duplication in the code base?

Reply:
We were already doing this, the class names were just identical so the auto-generation wasn't handling it properly. We already had tag/category subclasses for meal plans.

This part of the code is autogenerated though so maintenance isn't an issue